File: JavaScript/8 kyu/L1- Bartender- drinks-.js
function getDrinkByProfession(param) {
  return (
    {
      jabroni: "Patron Tequila",
      "school counselor": "Anything with Alcohol",
      programmer: "Hipster Craft Beer",
      "bike gang member": "Moonshine",
      politician: "Your tax dollars",
      rapper: "Cristal",
    }[param.toLowerCase()] || "Beer"
  );
}

File: Python/8 kyu/L1- Bartender- drinks-.py
def get_drink_by_profession(param):
    # code me!
    try:
        return {
            "jabroni": "Patron Tequila",
            "school counselor": "Anything with Alcohol",
            "programmer": "Hipster Craft Beer",
            "bike gang member": "Moonshine",
            "politician": "Your tax dollars",
            "rapper": "Cristal"
        }[param.lower()]
    except KeyError:
        return "Beer"
